File: single-language-analysis/dataset-files/java/TestSmell006.java
// Original URL: https://github.com/yuchting/yuchbox/blob/c4e3d7c87a47f192a4a7632fbb91d68fb3dbd719/server/src/weibo4j/WeiboTest.java#L276-L283

public class TestSmell006 {
	@Test
	public void testFavorites() throws WeiboException, InterruptedException {
		assertNotNull(weibo.getFavorites());
		assertNotNull(weibo.getFavorites(2));
		long id = 4052331047L;
		assertNotNull(weibo.createFavorite(id));
		Thread.sleep(1000);
		assertNotNull(weibo.destroyFavorite(id));
	}
}

File: single-language-analysis/dataset-files/python/test_smell_006.py
import pytest

class TestWeibo:
    def test_smell_006(self):
        assert weibo.get_favorites() is not None
        assert weibo.get_favorites(2) is not None
        id = 4052331047
        assert weibo.create_favorite(id) is not None
        import time
        time.sleep(1)
        assert weibo.destroy_favorite(id) is not None